### ISSUE TITLE
Error on SA disconnection

### ISSUE THREAD

**Issue by suzzlo** [jimmykane/quantified-self#164]
It's impossible to disconnect my SA account

![image](https://user-images.githubusercontent.com/105657/60031438-0dfd8700-96a5-11e9-9aed-aee34b5780fa.png)

Click "Disconnect"

![image](https://user-images.githubusercontent.com/105657/60031484-2372b100-96a5-11e9-9b25-35ed14273f5b.png)

Actually automatic import of each SA activity doesn't work for me. 

![image](https://user-images.githubusercontent.com/105657/60031689-7c424980-96a5-11e9-93ed-0c065aa28706.png)

I always get a notification about queued activities

![image](https://user-images.githubusercontent.com/105657/60031595-561ca980-96a5-11e9-8952-3a5016a129a3.png)

:-(

**Comment by NordicFlyer** [comment]
I discovered the same issue. Logging out of the application and logging back in (and disconnecting) seems to be a good workaround for now. I hope this helps.

**Comment by jimmykane** [comment]
Hey could you guys try the same at beta.quantified-self.io ? There is more logging in the console log.

**Comment by jimmykane** [comment]
Also please provide browser info . On chrome it works so far for example

**Comment by NordicFlyer** [comment]
Safari 12.1.1 on https://beta.quantified-self.io
Cannot authorize - stuck on wait window after filling out credentials

Chrome Version 75.0.3770.100 (Official Build) (64-bit)
login/authorize works; disconnect works

**Comment by jimmykane** [comment]
I think I know the issue with safari. Might be the popup blocking for some reason (Although the popup happens). Cannot reproduce it but I do have reports. The disconnect should work though. 

I cannot do something about that, rather to change all login procedures to redirect logic instread of popup. How would you feel about that?

**Comment by NordicFlyer** [comment]
The pop-up windows are not blocked for both sites on Safari. After multiple attempts I did get this message: 
![Screen Shot 2019-06-29 at 2 25 27 PM](https://user-images.githubusercontent.com/9980081/60389174-78823e80-9a7a-11e9-9108-1e25de0eed1c.png)

The disconnect did not work on Safari until I logged out and back in again (a day or two ago) Later attempt(s) today to reconnect also failed. A redirect may be more useful as many people really despise pop-ups of any kind.
